examples/blog/build/fakerest.js

```diff
@@ -0,0 +1 @@
+../../../node_modules/fakerest/dist/FakeRest.min.js
```

Comment:
It's just a symlink to the fakerest node module, to be able to import it in the `<script>` of the demo index.